Automattic/jetpack
# VideoPress: introduce PlaybackControl component

Fixes #

#### Changes proposed in this Pull Request:

* This is a refactoring PR with some tiny changes. 
  * Extract the code to create the new PlaybackControl component
  * Rename the panel title to Playback
  * Always show the autoplay control help
  * Use typescript

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to Block Editor
* Select/Add a VideoPress video block
* Open the sidebar
* Confirm the `Playback` panel is there
* Confirm it's possible to change the block attributes from there
* Confirm the changes are propagated to the player


https://user-images.githubusercontent.com/77539/200431856-8d85d03f-169b-4200-89d6-474dbf5f377a.mov
